src/dlgConnectionProfiles.cpp

```diff
@@ -1016,7 +1016,7 @@ void dlgConnectionProfiles::slot_item_clicked(QListWidgetItem* pItem)
             val = QStringLiteral("<center><a href='http://www.stickmud.com/'>stickmud.com</a></center>");
         }
         if (profile_name == QStringLiteral("Clessidra")) {
-            val = QStringLiteral("<center><a href='http://www.clessidra.it/'>clessidra.it</a></center>");
+            val = QStringLiteral("<center><a href='http://www.clessidra.it/'>http://www.clessidra.it</a></center>");
```

Comment:
You do not need the URL *scheme* there - it is, I think, customary to just show the `www.` in addition to the existing text, IMHO.

Reply:
Yes, but it's what they want and I don't mind.

Reply:
Also it's what most of other MUDs we have show (and they'd like consistency)

Reply:
Oh, yeah - apart from where I put in two for WoTMUD which are to two different domains and it was safer to high the URL details to avoid confusion, StickMUD and the Spanish "Reinos de Leyenda"... so forget my noise on this (mayhaps in the future I will whip 'em all {the scheme details} out but that will be a whole different PR. :smiling_imp:)